packages/comlink_language_server/src/documents.ts

```diff
@@ -1,12 +1,12 @@
-import { promises as fsp } from 'fs';
+import { promises as fsp } from 'node:fs';
 import {
   DocumentUri,
   TextDocumentContentChangeEvent,
 } from 'vscode-languageserver';
 
 import { ComlinkDocument } from './document';
 import { stripUriPrefix } from './lib';
-import { Diagnostic, Profile, ProfileSpans } from './parser/model';
+import { Diagnostic, Profile, ProfileSpans } from '@superfaceai/comlink';
```

Comment:
How does it work with linked package? It works thanks to bundling right?

Reply:
Yes, since this is basically a "binary" release (as opposed to a library), and I was using it this way during development. It's a good question whether we want to keep it this way or add a proper npm dependency.